snuba/datasets/configuration/replays/entities/aggregated.yaml

```diff
@@ -0,0 +1,343 @@
+      args:
+        {
+          func: min,
+          arg_types:
+            [{ type: UInt, args: { size: 16, schema_modifiers: [nullable] } }],
+        },
+    },
+    {
+      name: os_name,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: os_version,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: platform,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types: [{ type: String }, { type: UInt, args: { size: 8 } }],
+        },
+    },
+    { name: retention_days, type: UInt, args: { size: 16 } },
+    {
+      name: sdk_name,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: sdk_version,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: started_at,
+      type: AggregateFunction,
+      args:
+        {
+          func: min,
+          arg_types:
+            [{ type: DateTime, args: { schema_modifiers: [nullable] } }],
+        },
+    },
+    {
+      name: user,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: user_id,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
+            ],
+        },
+    },
+    {
+      name: user_name,
+      type: AggregateFunction,
+      args:
+        {
+          func: anyIf,
+          arg_types:
+            [
+              { type: String, args: { schema_modifiers: [nullable] } },
+              { type: UInt, args: { schema_modifiers: [nullable], size: 8 } },
```

Comment:
Is this for byte array? A single char?

Reply:
Its the data-type which contains the result of the `-If` condition in the materialized view.

Reply:
Oh.. my bad I totally wasn't thinking correctly 😓